lib/pxu_auth0/user_helpers.ex

```diff
@@ -3,10 +3,29 @@ defmodule PxUAuth0.UserHelpers do
   Conveniences for Auth0 authentication.
   """
 
+  @spec current_user(map) :: map | nil
   def current_user(conn), do: conn.assigns[:current_user]
 
+  @spec logged_in?(map) :: boolean
   def logged_in?(conn), do: Map.has_key?(conn.assigns, :current_user)
 
-  def in_group?(conn, group_name),
+  @doc """
+  Checks if logged-in user belongs to a group or is in one of the groups passed in.
+
+  iex> in_group?(conn, "some group")
+  true
+
+  iex> in_group(conn, ["group 1", "other group"])
+  true
+  """
+  @spec in_group?(map, String.t() | list) :: boolean
+  def in_group?(conn, group_name) when is_binary(group_name),
     do: conn |> current_user() |> Map.get(:groups, []) |> Enum.member?(group_name)
+
+  def in_group?(conn, groups) when is_list(groups) do
+    conn
+    |> current_user()
+    |> Map.get(:groups, [])
+    |> Enum.any?(fn x -> Enum.member?(groups, x) end)
```

Comment:
Heh, kinda surprised you didn't use the `&` shorthand here.

Reply:
I still find those hard to read sometimes

`Enum.any?(fn x -> Enum.member?(groups, x) end)`
vs
`Enum.any?(&Enum.member?(groups, &1))`
whats your pref?

Reply:
Yeah, I go back and forth. The longhand is definitely more readable. Should we use prefer the longhand, unless the shorthand will keep a piped function to a single line?

Reply:
Feels like a place we can use our judgement, if it obscures it and makes it harder to read then use the other?

Reply:
👍